examples/cloudflare-workers/src/worker.ts

```diff
@@ -0,0 +1,86 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first worker.
+ *
+ * - Run `npm run dev` in your terminal to start a development server
+ * - Open a browser tab at http://localhost:8787/ to see your worker in action
+ * - Run `npm run deploy` to publish your worker
+ *
+ * Learn more at https://developers.cloudflare.com/workers/
+ */
+
+class MomentoFetcher {
+	private readonly apiToken: string;
+	private readonly baseurl: string;
+	constructor(token: string, endpoint: string) {
+		this.apiToken = token;
+		this.baseurl = `https://api.cache.${endpoint}/cache`;
```

Comment:
Chatted with Matt on Slack but basically I think we should assume the user gets the REST endpoint from the docs or console, and they are copying and pasting it here. So instead of prepending `api.cache.` to the parameter passed in, we should just use the parameter as is